fe/src/components/user/campaigns.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div
+    class="mt-6 grid grid-cols-3 gap-6 sm:grid-cols-1 lg:grid-cols-2 xl:grid-cols-3 campaigns"
+  >
+    <CampaignCard
+      v-for="i in 6"
+      :key="i"
+      :title="title"
+      :description="description"
+      :eth-value="ethValue"
+      :img-src="imgSrc"
+      :days-left="daysLeft"
+    ></CampaignCard>
+  </div>
+  <div class="w-full mt-8 h-14 flex items-center justify-center">
+    <BasePaginator
+      :current-page="currentPage"
+      :last-page="10"
+      :per-page="first"
+      :on-page-change="setPage"
+    />
+  </div>
+</template>
+<script setup lang="ts">
+import CardSample from "~/mocks/card-sample.json";
+import CampaignCardProps from "~/types/CampaignCardProps";
+import UserCardSample from "~/mocks/user-card-sample.json";
+import UserCardProps from "~/types/UserCardProps";
+
+const userCardValueSample = ref<UserCardProps>(UserCardSample);
+
+const first = ref(5);
+const currentPage = ref(1);
+const cardValueSample = ref<CampaignCardProps>(CardSample);
+const { imgSrc } = userCardValueSample.value;
+
+const { title, description, ethValue, daysLeft } = cardValueSample.value;
+const setPage = (first: number, page: number) => {
+  currentPage.value = page;
+  console.log(first);
+};
```

Comment:
what is this variable and its purpose `first`
can we give meaning variable name?
```suggestion
const setPage = (first: number, page: number): void => {
  currentPage.value = page;
};
```

Reply:
overlooked. Updating it to proper naming.